doc/source/examples/multiphysics/rayleigh-benard-convection/rayleigh-benard-convection.rst

```diff
@@ -45,15 +45,16 @@ The incompressible Navier-Stokes equations with a Boussinesq approximation for t
 
 where :math:`\beta` and :math:`T_0` denote thermal expansion coefficient and a reference temperature, respectively.
 
-A two-dimensional block of fluid is heated from its bottom wall at :math:`t = 0` s. The temperature of the bottom wall is equal to :math:`T_h=50`, the temperature of the top wall is equal to :math:`T_c=0`, and the left and right walls are insulated. By heating the fluid from the bottom wall, the buoyant force (natural convection) creates vortices inside the fluid. The shape and number of these vortices mainly depend on the Rayleigh number [1, 2]:
+A two-dimensional block of fluid is heated from its bottom wall at :math:`t = 0` s. The temperature of the bottom wall is equal to :math:`T_\text{h}=10`, the temperature of the top wall is equal to :math:`T_\text{c}=0`, and the left and right walls are insulated. By heating the fluid from the bottom wall, the buoyant force (natural convection) creates vortices inside the fluid. The shape and number of these vortices depend on the Rayleigh number and the Prandtl [#ouertatani]_ [#venturi2010]_ :
 
     .. math::
```

Comment:
Due to indentation, the equation are outlined as if they a citation or something (see capture)
![image](https://github.com/user-attachments/assets/cdc098a3-ed34-4637-8bdb-27ca354fb009)

To correct the indentations on lines 50-52:

> .. math::
>     Ra &= \frac{\rho^2 \beta g (T_\text{h} - T_\text{c}) H^3 c_\text{p}}{k \mu} \\
>     Pr &= \frac{\mu c_\text{p}}{k}

Reply:
Can you also fix the indentation for lines 39-44 please, so that other people don't get the same issue if they reference to this example while making another:

> 
> The incompressible Navier-Stokes equations with a Boussinesq approximation for the buoyant force are:
> 
> .. math::
>     \nabla \cdot {\bf{u}} = 0
> 
> .. math::
>     \rho \frac{\partial {\bf{u}}}{\partial t} + \rho ({\bf{u}} \cdot \nabla) {\bf{u}} = -\nabla p + \nabla \cdot {\bf{\tau}} - \rho \beta {\bf{g}} (T - T_0)